docker/alpine-3.5-repositories

```diff
@@ -0,0 +1,2 @@
+http://dl-cdn.alpinelinux.org/alpine/v3.5/main
```

Comment:
Good catch! Nothing, anymore. It used to be for forcing npm's alpine image from 3.4 to 3.5, but nothing uses npm's image now.